tests/integration/qbs/test_qbs_change_answers.py

```diff
@@ -0,0 +1,72 @@
+
+from tests.integration.integration_test_case import IntegrationTestCase
+
+
+class QbsHappyPath(IntegrationTestCase):
+
+    def test_happy_path_2_0001(self):
+        self.happy_path('2', '0001')
+
+    def happy_path(self, eq_id, form_type_id):
+        self.launchSurvey(eq_id, form_type_id, roles=['dumper'])
+
+        # We are on the landing page
+        self.assertInPage('>Start survey<')
+        self.assertInPage('Quarterly Business Survey')
+
+        # We proceed to the questionnaire
+        self.post(action='start_questionnaire')
+
+        # We are in the Questionnaire
+        self.assertInPage('>Quarterly Business Survey</')
+        self.assertInPage('On 1 April 2016 what was the number of employees for Integration Tests?')
+        self.assertInPage('>Save and continue<')
+
+        # When I submit an answer
+        self.post(action='start_questionnaire')
```

Comment:
removed the extra unnecessary post.. not sure yet why it still passes with an extra post.